src/features/testing/assistant.md

```diff
@@ -1 +1,38 @@
-# Assistant (WIP)
+# Assistant
+
+> **PRO FEATURE**
+
+The Assistant page is a feature offered for **Pro users**.
+It helps you understand requests, explains elements like headers and suggests attack vectors.
+
+The AI assistant in Caido is taylored for security research.
+
+Data is sent to a third party (OpenAI) and can be stored for up to 30 days.
+
+Make sure to anonymize sensitive information when you use this tool.
+
+## Credits system
+
+To prevent abuse, we use a credit system.
+
+Credit usage depends on the model used (currently only ChatGPT 3.5).
```

Comment:
Some more information:
- Each pro user gets 500 000 credits per month
- Lets add a example of a Request (in text format) and the number of credits it represents
- The 1 credit = 1 token is only valid for the gpt-3.5-turbo model, we will add support for other models eventually
- I would explain a bit what models are and LLM (large language models) in general. More specifically around limits of it like hallucination and that is not going to "hack for your"

You might have to do some reading around chatgpt and LLM to explain that last bit, no need to go in super details but the basic would be good to have in here.